hbw/app/javascript/packs/hbw/components/form/select.js.jsx

```diff
@@ -232,8 +232,20 @@ modulejs.define('HBWFormSelect',
           clearTimeout(timer);
           timer = setTimeout(functionCall, ms);
         };
+      },
+
+      getChoices (value) {
+        if (this.props.params.mode === 'select') {
+          this.addCurrentValueToChoices(value);
+
+          return this.props.params.choices.slice();
+        } else if (this.props.params.mode === 'lookup') {
+          return this.props.params.choices;
+        }
```

Comment:
```
else {
  return null;
}
```

Reply:
It is case like this, isn't it?
```
// bad
function foo() {
  if (x) {
    return x;
  } else {
    return y;
  }
}

// good
function foo() {
  if (x) {
    return x;
  }

  return y;
}

```
https://github.com/airbnb/javascript#blocks--no-else-return

Reply:
It is, but it is not our style :)
https://github.com/latera/homs/blob/5e4fba2a0497145959cfa34117fc9f86e4e2b03e/.eslintrc#L25

Reply:
But it's not mentioned in https://github.com/latera/homs/blob/master/CONTRIBUTING.md

Reply:
Ok: https://github.com/latera/homs/pull/208

Reply:
Nice

Reply:
Thanks all, that's awesome now.